index.js

```diff
@@ -18,20 +18,37 @@ bedrock.events.on('bedrock-cli.init', () => bedrock.program
   .option('-h, --hostname <value>', 'The ledger host.')
   .option('-k, --ignoreSslErrors', 'Use strict SSL.')
   .option('-s, --send', 'Send a demonstration operation.')
+  .option('-r, --recordPath <value>', 'A path to a json record to be sent.')
   .option('-g, --getRecordId <value>', 'Get a record by ID.')
 );
 
 bedrock.events.on('bedrock.started', async () => {
   try {
     await work();
   } catch(e) {
+    console.error(
+      'event bedrock.started failed when trying to initialize work.');
     console.error(e.message);
   }
   bedrock.exit();
 });
 
+// gets or creates a default record
+function getRecord(path) {
+  if(path) {
+    // must have extension .json or .js
+    return require(path);
+  }
+  return {
+    '@context': constants.TEST_CONTEXT_V1_URL,
+    id: `https://example.com/transaction/${uuid()}`,
+    price: Math.floor(Math.random() * 1000000000000),
+  };
+}
+
 async function work() {
-  const {getRecordId, hostname, ignoreSslErrors, send} = bedrock.program;
+  const {getRecordId, recordPath,
```

Comment:
fix style, probably line break at the equal sign.

Reply:
Addressed in https://github.com/digitalbazaar/bedrock-ledger-test-cli/pull/1/commits/e46fb1ced35923628ce105d70f27de6ecf2bd4cd